README.md

```diff
@@ -337,6 +344,90 @@ spec:
 - The operator fetches the secret from GCP Secret Manager and injects it as a
 secret:
 
+```shell
+% make install
+% make deploy
+% kubectl get secret externalsecret-operator-externalsecret-sample -n externalsecret-operator-system \
+  -o jsonpath='{.data.example-externalsecret-key}' | base64 -d
+```
+
+<a name="gitlab-cicd-variables"></a>
+
+## Gitlab CI/CD Variables
+
+<a name="gitlab-cicd-variables-pre"></a>
+
+#### Prerequisites
+- A Gitlab project with a CI/CD variable, who's key is `example-externalsecret-key`
+- A [Gitlab personal access token](https://gitlab.com/-/profile/personal_access_tokens)
```

Comment:
When creating a Gitlab Personal Access Token, I’m presented with Scope options for the key, such as `api`, `read_user` etc.

Could you document which one should be enabled, preferably the less permissive one.

Reply:
will do. I think API is enough

Reply:
I put `read_api` in the doc, but I still have to verify that it's enough.

Reply:
I think that should be it too!

Reply:
Let me know when u change it so we can merge this code :)

Reply:
@jonatasbaldin read_api works. :+1: